### DISCUSSION TITLE
keri-dev 07/20/2023

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#38] [Meetings]
Recording:
https://us06web.zoom.us/rec/share/9RtKAuTNe1417D-4tgdLzmdsrRz63EuaBOysMQU4EZ0ysw4aaZXsIXo1tIRNdzyC.FJhPr84fMxOsGoQN

Passcode: Z0JP84J+

**keria**
PR #67 - some test in credentialing failing
PR #78 - needs review
Delegated Multisig with Tholder impl

For codecov:
- Change once we hit 90 overall
- Patch 75%?

**signifypy**
Delegated Multisig with Tholder impl
Minor fixes from PR
PR #45 - Draft

**signify-ts**
Typescript clean up
Revocation PR
Delegated Multisig with Tholder impl

**keripy**
Delegated Multisig with Tholder impl

**cesride**
no major update

**parside**
PR coming from Jason with new examples

**Questions**
vLEI Ecosystem - QVI Delegation, Multiple QVI credentials.

_Sending the path in sig headers(?)_
Path used to solve credentials within credentials for partial signing, path (SAD path language) is to the portion they signed.
Also, used for forwarding in KERI embedded messages - see video discussion.

We don't sign our credentials, you shouldn't either!

_How are we going to migrate KELs moving forward?_
We'll work on a migration tool, it is TBD right now, but we acknowledge the need.

**Demo**
Delegated Multisig - Starts around 28 minutes.

**Comment by nkongsuwan** [comment]
If delegation is not allowed for a QVI AID within the vLEI ecosystem, is the following approach allowed for a QVI?
1. Incept a new AID that is different from the one used as the QVI AID
2. Use QVI AID to issue a Legal Entity vLEI to the new AID, i.e., the QVI issues Legal Entity vLEI to itself using the new AID
3. Use the new AID to delegate
4. Use the delegated AID to issue credentials.
